### PR TITLE
Handle gracefully config path not existing (Fixes #238)

### LINKED ISSUE #238: `.config` doesn't always exist
In fresh WSL `topgrade` fails with:
```
ERROR topgrade::config > Unable to write the example configuration file to /home/mateusz/.config/topgrade.toml: No such file or directory (os error 2)
Error: Error reading the configuration
Caused by: No such file or directory (os error 2)
```
This is caused by non existent `~/.config`, creating this directory fixes the issue but I think `topgrade` could print better error or create directory itself.

### PR DESCRIPTION
**Pull request by idanski**
Note: removed verbosity from config file, as failing while reading the config file needs a logger output and env_logger can't be init twice or changed after init as far as I could tell.

@r-darwish WDYT?

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -64,19 +64,18 @@ pub struct ConfigFile {
     yay_arguments: Option<String>,
     no_retry: Option<bool>,
     run_in_tmux: Option<bool>,
-    verbose: Option<bool>,
```
Comment (r-darwish) [inline src/config.rs]: We don't have to get rid of this. We could just perform two ifs, one before the configuration is loaded, which checks `opt.verbose`, and one after that checks `config.verbose`

File: src/config.rs [hunk 2]
```diff
@@ -120,6 +119,26 @@ impl ConfigFile {
             .context(ErrorKind::Configuration)?;
         Ok(())
     }
+
+    /// Creates a new, blank config file.
```
Comment (r-darwish) [inline src/config.rs]: We can probably derive `Default` instead of implementing a method.

File: src/config.rs [hunk 3]
```diff
@@ -64,19 +64,18 @@ pub struct ConfigFile {
...
     run_in_tmux: Option<bool>,
-    verbose: Option<bool>,
     cleanup: Option<bool>,
     only: Option<Vec<Step>>,
 }
 
 impl ConfigFile {
     fn ensure(base_dirs: &BaseDirs) -> Result<PathBuf, Error> {
-        let config_path = base_dirs.config_dir().join("topgrade.toml");
+        let config_path = base_dirs.config_dir().join("/plooopy/topgrade.toml");
```
Comment (r-darwish) [inline src/config.rs]: I suppose you did that for debugging

File: src/config.rs (hunk 1 above)
Comment (idanski) [reply src/config.rs]: hmm, but I want to call `builder.init()` before running config `load`, after running init() is there any way to update the logger?

Comment (idanski) [reply src/config.rs]: I think the answer is no, unless you have another idea..?

https://github.com/seanmonstar/pretty-env-logger/issues/27

Comment (r-darwish) [reply src/config.rs]: Yeah, you're correct.

**Pull request merged**